vignettes/rhino-fluent.Rmd

```diff
@@ -0,0 +1,556 @@
+---
+title: "Tutorial: Build a Rhino application using shiny.fluent"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Tutorial: Build a Rhino application using shiny.fluent}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+## 1. Hello World with `rhino` and `shiny.fluent`
+
+### Getting started with our development environment
+
+Let us first install and create our `rhino` project structure.
+
+To install `rhino`, please run the following on a R console in your working directory
+
+```         
+install.packages("rhino")
+```
+
+Now that `rhino` is installed, let us use it to initiate a project which will give us a structure
+
+```         
+rhino::init()
+```
```

Comment:
This is misleading. The most common case is that the user will use RStudio, thus they should use the Project  Wizard. There is no point in describing the process of creating a new project in detail, but you should mention all options and point to the documentation (main Rhino tutorial) where the user can find those details.